UnityProject/Assets/Scripts/Objects/Construction/APCFrame.cs

```diff
@@ -302,7 +302,7 @@ private void WrenchedStateInteraction(HandApply interaction)
 
 				econs.Pool();
 
-				GameObject WallMount = Spawn.ServerPrefab(APCObject, gameObject.GetComponent<CustomNetTransform>().ServerPosition, interaction.Performer.transform.parent, spawnItems: false).GameObject;
+				GameObject WallMount = Spawn.ServerPrefab(APCObject, gameObject.GetComponent<UniversalObjectPhysics>().transform.position, interaction.Performer.transform.parent, spawnItems: false).GameObject;
```

Comment:
Don't need to get the component if we're just using it for `transform.position`.